rust_src/remacs-sys/lib.rs

```diff
@@ -208,6 +208,14 @@ pub enum Symbol_Interned {
     InternedInInitialObarray = 2,
 }
 
+#[repr(C)]
+pub enum Symbol_Redirect {
+    Plainval = 4,
+    Varalias = 1,
```

Comment:
Rust enums are generally camelcase, so I'd be tempted to go with `PlainVal` and `VarAlias`.

Reply:
Good catch, updated.